### PR TITLE
[SC-57] Gnosis Chain Deploy

### PR DESCRIPTION
**Pull request by hexonaut**
Gnosis Chain Deployment.

### CODE REVIEW

File: test/deploys/100/primary/SparkDeploy_20230714_GnosisPrimary.t.sol [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import "../../../SparkDeployBase.t.sol";
+
+contract SparkDeploy_20230714_GnosisPrimaryTest is SparkDeployBaseTest {
+    constructor() {
```
Comment (lucas-manuel) [inline test/deploys/100/primary/SparkDeploy_20230714_GnosisPrimary.t.sol:7]: New line to match change in other test

File: validate-deploy.sh [hunk 2]
```diff
@@ -1,23 +1,48 @@
...
 if [ -z $1 ]; then
-  echo "Usage: ./validate-deploy.sh <broadcast-file>"
+  echo "Usage: ./validate-deploy.sh <broadcast-file>[ --local]"
   exit 1
 fi
 
 # load the json file
 json=$(cat $1)
 
+# setup variables based on environment
```
Comment (lucas-manuel) [inline validate-deploy.sh:15]: Lets update all comments in this file to sentence case to be consistent

**Review by lucas-manuel — CHANGES_REQUESTED** [review]
LGTM except I can't get pool to verify

File: test/deploys/100/primary/SparkDeploy_20230714_GnosisPrimary.t.sol (hunk 1 above)
Comment (hexonaut) [reply test/deploys/100/primary/SparkDeploy_20230714_GnosisPrimary.t.sol:7]: fixed

File: validate-deploy.sh (hunk 2 above)
Comment (hexonaut) [reply validate-deploy.sh]: done

**Comment by hexonaut** [comment]
Going to add a deploy for the Gnosis testnet as well.

**Comment by hexonaut** [comment]
Added testnet. Waiting on governance relay for both.

**Comment by hexonaut** [comment]
Ok this is ready for review now with final deploy.

**Review by barrutko — APPROVED** [review]

**Review by lucas-manuel — APPROVED** [review]

**Pull request merged**